packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -117,7 +118,12 @@ const webpackConfig = {
         to: `${I18N_DIR}/dist/properties/`,
       }
     ]),
-
+    new ReplacePlugin({
```

Comment:
Let's also add a comment above this referencing another JIRA (may need to create one) to address this at the Courage framework level.